.gitignore

```diff
@@ -45,6 +45,7 @@ build/
 .gradle
 local.properties
 *.iml
+gen/
```

Comment:
just curious, what is generating this?

Reply:
I don't know either, after build android with `gradle assemble`, it created there files: `buildConfig.java`, `Manifest.java` and `R.java`. On each of them there is comments said: 

```
/*___Generated_by_IDEA___*/
```

So I just add them there.